litmus-portal/namespaced-k8s-template.yml

```diff
@@ -126,10 +126,6 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-            - name: PORTAL_SCOPE
-              value: "namespace"
-            - name: PORTAL_ENDPOINT
```

Comment:
We need this env for namespace mode

Reply:
But why do we need this exactly?

Reply:
For namespace mode we ask PORTAL_ENDPOINT

Reply:
This may not be required.

Reply:
Reverted this change.